CHANGELOG.md

```diff
@@ -14,6 +14,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ### Added
  - pipe
 
+### Changes
+ - takeSorted/asyncTakeSorted moved optional comparator argument to the beginning
```

Comment:
I recommend "optional comparator is now the first argument". Also I think you should also call out all the other arguments that can no longer be undefined here. My goal for this file is that if we change a method its name is in this file, such that if I search changelog for a method name I will know which releases have note-worthy changes to it.

Reply:
done